src/VisualStudio/Core/Def/Implementation/VirtualMemoryNotificationListener.cs

```diff
@@ -21,6 +22,9 @@ namespace Microsoft.VisualStudio.LanguageServices
     [Export, Shared]
     internal sealed class VirtualMemoryNotificationListener : ForegroundThreadAffinitizedObject, IVsBroadcastMessageEvents
     {
+        // memory threshold to turn off full solution analysis - 200MB
+        private const long MemoryThreshold = 200 * 1024 * 1024;
```

Comment:
Do we handle 32-bit processes on 64-bit machines?  We actually get 4GB of VM on 64-bit machines, and VS is large address aware.  Are we triggering this at 1.5GB even on 64-bit machines?